frontendUiRootImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/VcsRootComboBox.java

```diff
@@ -36,7 +55,13 @@ public VcsRootComboBox(@MinLen(1) List<GitRepository> repositories) {
 
   @Override
   @UIEffect
-  public void updateRepositories(@MinLen(1) List<GitRepository> repositories) {
+  public void updateRepositories() {
+    // GitUtil.getRepositories(project) should never return empty list because it means there's no git repository in an opened
```

Comment:
Is this comment actually true in the light of #255?

Reply:
removed and todo left